subset/dot1x/dot1x_config.yaml

```diff
@@ -0,0 +1,9 @@
+radius:
```

Comment:
Huh -- so my previous comment seems to have been lost (don't see it anywhere).  What' the intent of this configuration?  Is there a reason it's not using the standard module configuration mechanism?  If this is "the ability to configure the system to use an external radius server" then you shouldn't be rolling your own capability for that... (see the ipaddr module for how it's configured).